db/re/item_db_usable.yml

```
@@ -45510,6 +45510,18 @@ Body:
     Type: Usable
     Script: |
       /* TODO */
+  - Id: 22566
+    AegisName: Frost_Crystal
+    Name: Last Year's Frost
+    Type: Usable
+    Buy: 20
```

the sell price is 5 according to DP
```suggestion
    Buy: 10
```